tests/Doctrine/DBAL/Migrations/Tests/Stub/Functional/MigrateNotTouchingTheSchema.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Doctrine\DBAL\Migrations\Tests\Stub\Functional;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class MigrateNotTouchingTheSchema extends AbstractMigration
```

Comment:
Good point: didn't know this, although it evidences the lack of a collaborator there (to be fixed another time, if you think it is relevant).